themes/rhds/layouts/partials/eddl/page-load-started.html

```diff
@@ -0,0 +1,60 @@
+<script>
+    window.appEventData = window.appEventData || [];
+
+    // Determine authors
+    let authors = '';
+    {{ if isset .Params "authors" }}
+      {{ range $index, $author := .Params.Authors }}
+        {{ if eq $index (sub (len $.Params.Authors) 1) }}
+            authors += '{{ $author }}';
+        {{ else }}
+            authors += '{{ $author }},'
+        {{ end }}
+      {{ end }}
+    {{ end }}
+
+    // Determine published date
+    let publishDateMachine = '';
+    {{ if isset .Params "date" }}
+      publishDateMachine = '{{ .PublishDate | time.Format "2006-01-02"}}';
+    {{ else }}
+      publishDateMachine = '';
+    {{ end }}
+
+    // Determine pageName, siteName, and subsections.
+    const siteName = 'RH|cloud'
+    const pathname = new URL(window.location.href).pathname;
+    const subsections = pathname.split('/');
+    let pageNameCategories = subsections.join('|');
+    if (pageNameCategories.endsWith('|')) {
+      pageNameCategories = pageNameCategories.substring(0, pageNameCategories.length-1);
+    }
+
+    const pageData = {
+      analyticsTitle: '',
+      blogAuthor: authors,
+      contentID: '',
+      contentType: '',
+      contentPublishDate: publishDateMachine,
+      dataObject: 'appEventData',
+      destinationURL: window.location.href,
+      gated: 'false',
+      pageCategory: (subsections[1]) ? subsections[1]: '',
```

Comment:
This is usually the first path in the route. Since we split the string with a leading slash (`/some/path`) the first section part should always be at index 1.